project/PekkoDependency.scala

```diff
@@ -114,11 +116,12 @@ object PekkoDependency {
               ep.toInt,
               maj.toInt,
               min.toInt,
-              Option(tagNumber).map(_.toInt),
+              Option(tagNumber).map(_.toInt).getOrElse(Integer.MAX_VALUE),
```

Comment:
Why was this change needed?

Reply:
yes - this is related to `adjusts the tagNumber logic to treat build numbers without an RC or M tag as newer than ones that have them`

Reply:
Okay, but how is `Integer.MAX_VALUE` going to provide a correct value here, or am I missing something? Is it just some sentinel value that will get ignored later (if so add a comment because logic/reasoning behind it is unclear).

Reply:
Oh nvm, its for ordering and not the actual value. @pjfanning Can you just add a comment explaining that its there to fix ordering and then ill approve PR.

Reply:
The values are used only in sorting.

* `-RC1-` will eval as 1
* no `-RC` or `-M` will now eval as MAX-INT
* MAX-INT > 1 so sorting will treat non-RC/non-milestone releases as newer

Old logic:
* `-RC1-` will eval as `Some(1)`
* no `-RC` or `-M` will eval as `None`
* `None` < `Some(1)` so sorting will treat non-RC/non-milestone releases as older

Reply:
Understood, just add a comment in the code quickly explaining this because on first glance its misleading.